app/assets/javascripts/discourse/tests/acceptance/sidebar-user-test.js

```diff
@@ -7,6 +7,7 @@ import {
   query,
   updateCurrentUser,
 } from "discourse/tests/helpers/qunit-helpers";
+import "discourse/routes/wizard";
```

Comment:
Because the wizard route is not guaranteed to be in the main bundle, tests that want to have it loaded needs to do this.